pkgs/development/tools/poetry2nix/poetry2nix/overrides/build-systems.json

```diff
@@ -4816,9 +4816,6 @@
     "setuptools",
     "setuptools-scm"
   ],
-  "filemagic": [
-    "setuptools"
-  ],
```

Comment:
poetry2nix is managed outside of nixpkgs (yes, I know) and we shouldn't touch and to an extend don't care to much about it.

Reply:
Cool, got it and will keep in mind for next time.